airflow/ui/src/dagsList.tsx

```diff
@@ -110,53 +74,67 @@ const columns: ColumnDef<DAG>[] = [
       </HStack>
     ),
     enableSorting: false,
+    header: () => "Tags",
   },
 ];
 
-const QuickFilterButton = ({ children, ...rest }: ButtonProps) => (
-  <Button
-    borderRadius={20}
-    fontWeight="normal"
-    colorScheme="blue"
-    variant="outline"
-    {...rest}
-  >
-    {children}
-  </Button>
-);
-
 const PAUSED_PARAM = "paused";
 
-export const DagsList = () => {
-  const cardView = false;
+// eslint-disable-next-line complexity
```

Comment:
Had to do this to keep the PR smaller, but the complexity of this function should be reduced.